README.md

```diff
@@ -95,29 +119,22 @@ The process for compiling Hybrasyl is detailed below.
 1. Install
    [Microsoft Visual Studio](https://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs.aspx).
    The Community Edition is free and capable of compiling all the needed projects (server, launcher).
+
 2. Clone the [launcher](https://github.com/hybrasyl/launcher) and 
    [server](https://github.com/hybrasyl/server) repositories to your local machine
    using a [git client](https://git-scm.com/downloads/guis), or with Visual Studio's built-in integration. 
    Make sure you clone them into separate directories. 
+
 3. Open the Hybrasyl Server solution (`Hybrasyl.sln`) in Visual Studio and
    update all NuGet packages (just building it will do this). The SDK for
    XML is now included in server to make this process (as well as making changes) 
    easier.
```

Comment:
alternatively if running `dotnet` commands you can use `dotnet restore` in the same directory as the `.csproj` file.

Reply:
added